src/main/java/com/iexec/worker/config/PublicConfigurationService.java

```diff
@@ -55,10 +57,10 @@ public ConfigServerClient configServerClient() {
                 configServerURL);
     }
 
-    @Bean
-    public ResultProxyClient resultProxyClient() {
-        return ResultProxyClientBuilder.getInstance(
-                Logger.Level.NONE,
-                publicConfiguration.getResultRepositoryURL());
+    public ResultProxyClient createProxyClientFromURL(String url) {
```

Comment:
I think we prefer to be as precise as possible here.
A proxy is far to generic, many things can be proxied in computer science
```suggestion
    public ResultProxyClient createResultProxyClientFromURL(String url) {
```